ipa-core/src/query/runner/hybrid.rs

```diff
@@ -120,6 +120,8 @@ where
         )
         .await?;
 
+        unimplemented!("protocol::hybrid::hybrid_protocol is not fully implemented");
```

Comment:
temporary addition to identify stall location. adding this just above line 116 allows the tests to pass